Userland/Libraries/LibC/dirent.cpp

```diff
@@ -72,8 +72,13 @@ static void create_struct_dirent(sys_dirent* sys_ent, struct dirent* str_ent)
     str_ent->d_off = 0;
     str_ent->d_reclen = sizeof(struct dirent);
 
-    int size = min((sys_ent->namelen + 1) * sizeof(char), sizeof(str_ent->d_name));
-    [[maybe_unused]] auto n = strlcpy(str_ent->d_name, sys_ent->name, size);
+    VERIFY(sizeof(str_ent->d_name) > sys_ent->namelen * sizeof(char));
```

Comment:
ˋsizeof(char)ˋ is always 1, so that’s probably redundant.

Reply:
Everyone loves to tell me this 😄, it's just a habit for clarity... but i'm not attached to it.

Reply:
fixed